docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -62,6 +65,47 @@ putting new dynamic templates through the <<indices-put-mapping, put mapping>> A
 all existing templates are overwritten. This allows for dynamic templates to be
 reordered or deleted after they were initially added.
 
+[[dynamic-mapping-runtime-fields]]
+==== Mapping runtime fields in a dynamic template
+If you want {es} to dynamically map new fields of a certain type as runtime
+fields, set `dynamic:runtime` in the index mappings. These fields are not
+indexed, and are loaded from `_source` at query time.
+
+Alternatively, you can use the default dynamic mapping rules and then create
+dynamic templates to map specific fields as runtime fields. You set
+`dynamic:true` in your index mapping, and then create a dynamic template to map
+new fields of a certain type as runtime fields.
+
+Let's say you have data where each of the fields start with `ip_`. Based on the
```

Comment:
shall we specify here that this example applies to both dynamic:true or dynamic:runtime?

Reply:
Good call. I think we can split the explanation into two paragraphs like this:

>Let's say you have data where each of the fields start with `ip_`. Based on the
<<match-mapping-type,dynamic mapping rules>>, {es} maps any `string` that passes
`numeric` detection as a `float` or `long`. However, you can create a dynamic
template that maps new strings as runtime fields of type `ip`.

>The following request defines a dynamic template named `strings_as_ip`. When
{es} detects new `string` fields matching the `ip*` pattern, it maps those
fields as runtime fields of type `ip`. You can use this template with either
`dynamic:true` or `dynamic:runtime`.

Reply:
Last sentence, shall we adapt it to something like "You can use this template with either dynamic:true or dynamic:runtime, as ip fields don't get dynamically mapped with either of the two". I am sure you will prettify it but that would be the message ;)